proto/process/agent.proto

```diff
@@ -603,6 +611,21 @@ message DNSEntry {
   repeated string names = 1;
 }
 
+message Cluster {
+	int32 nodeCount = 1;
+	map<string, int32> kubeletVersions = 2;
+	map<string, int32> apiServerVersion = 3;
```

Comment:
is it possible to have multiple API server versions? (hence is the map necessary? Also if possible should be in plural form like above

Reply:
yes, potentially we could have multiple api versions. 
https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew
```
In highly-available (HA) clusters, the newest and oldest kube-apiserver instances must be within one minor version.
```

Let me fix the naming. 🙇‍♂️

Reply:
The idea was to start to only collect the main apiVersion but have the possibility to extend that.